src/styles/card.scss

```diff
@@ -0,0 +1,73 @@
+.card {
+  width: 198px; // including border
+  height: 406px;
+  border: 1px solid $card-border-color;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+
+  &__image {
+    padding: 32px 19px 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 150%;
+    color: $card-title-color;
+    padding-bottom: 4px;
+    padding-left: 16px;
+    padding-right: 16px;
+  }
```

Comment:
Repeated padding values for `padding-left` and `padding-right` can be combined into a single shorthand property. For instance, `padding-left: 16px; padding-right: 16px;` could be simplified to `padding: 0 16px;` if no padding-top or padding-bottom is needed, or included in a full shorthand if they are.